### PR TITLE
Add Reference to IMAGE_DIFF_DIR to ReadMe

### PR DESCRIPTION
**Pull request by pres**

### CODE REVIEW

**Comment by Grubas7** [comment]
@pres Nice! I'm using Snapshots for a long time and I didn't know about this feature.

File: README.md [hunk 1]
```diff
@@ -49,7 +49,9 @@ Installation with CocoaPods
 |Name|Value|
 |:---|:----|
 |`FB_REFERENCE_IMAGE_DIR`|`$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/ReferenceImages`|
+|`IMAGE_DIFF_DIR`|`$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/FailureDiffs`|
 
+Define the `IMAGE_DIFF_DIR` to the directory where you want to store diffs of failed snapshots.
```
Comment (Grubas7) [inline README.md]: Maybe move this few lines above and mark as point 3. Additional info about optional nature of this macro (and maybe default value) would be helpful.

**Comment by nscoding** [comment]
thank you for this, can you sign the `License Agreement` so I can merge this PR?

**Comment by pres** [comment]
- [x] CLA

**Pull request merged**